backend/src/components/utils.js

```diff
@@ -135,6 +134,44 @@ const utils = {
       log.error('filterWebAdeDependencies', 'Error in supplied webade configuration list');
       throw new Error('Unable to fetch dependencies - Error in supplied webade configuration list');
     }
+  },
+
+  /**
+  * Is this call allowed to be made for the acronym it's being made for?
+  * @param {string} token - The user's token from the request.
+  * @param {string} acronym - The app acronym.
+  * @returns {string} Error Message - Undefined if permitted, a string of the error message to return if not permitted.
+  */
+  checkAcronymPermission: (token, applicationAcronym) => {
```

Comment:
Should we just go to the DB for all acronym/app-based security, only use KeyCloak for GETOK access and permissions?

Reply:
Discussed this morning